lib/commands/context.js

```diff
@@ -330,6 +330,9 @@ extensions.onPageChange = async function (pageChangeNotification) {
         await this.remote.pageLoad();
       }
 
+      logger.debug("Clearing any frames");
+      this.curWebFrames = [];
```

Comment:
I'm not sure this is right.
We do need to understand what the different conditional branches are for.
I think there are three main ones.
- We have loaded a new page
- The window is closed
- We have reloaded the current page OR followed an anchor link (as hinted by the comment in that branch).
The fix is in the third branch, so if it navigates to a page anchor will it reset the frame? Presumably we don't want that.
Then what if the page loads completely to a new page, not a reload. Then we go to the first branch, and would expect to get the same StaleElementReferenceError error.